src/main/eo/org/eolang/math/number.eo

```diff
@@ -155,7 +158,41 @@
         value.neg
 
   # Make ^.n power x
-  [x] > pow /?
+  [x] > power
```

Comment:
@maxonfjvipon I think we should not remove the `pow` object until the `power` object is stable in use.

Reply:
done